modules/ROOT/pages/security.adoc

```diff
@@ -0,0 +1,252 @@
+
+|===
+
+
+== Compatibility
+
+Server Compatibility::
++
+--
+By default the server scheme is specified as `couchbases://`.
+
+You can set the {bootstrap-schema--xref--bootstrap-server-tls-skip-verify} flag `true` to connect to 'default CBS'.
+But, this must only be done for testing or development purposes. 
+--
+
+Client Compatibility::
+Couchbase Lite client applications must be updated to use TLS when connecting to {sgw} nodes running in default mode.
+
+
+
+== Behavior
+
+The content in <<tbl-route-behavior>> shows the communications behavior between Sync Gateway and communication endpoints.
+
+
+.Sync Gateway behavior
+[#tbl-route-behavior, cols="4a,6a,4,6a, options="header"]
+|===
+
+.>h|Endpoint
+.>h|Required Config Items
+.>h|TLS Opt-Out Trigger
+.>h|Behavior
+
+|REST API, Couchbase Lite and Inter-sync gateway replication
+|{bootstrap-schema--xref--api-https-tls-cert-path} +
+{bootstrap-schema--xref--api-https-tls-key-path}
+|Provide nil values for the key/certificate paths
+|TLS encryption is enabled by default, unless the opt-out is triggered
+
+|Couchbase Server
+|Use `couchbases://` server scheme +
+Set the certificate or key path:
+
+* {bootstrap-schema--xref--bootstrap-ca-cert-path} +
+* {bootstrap-schema--xref--bootstrap-certpath} +
+* {bootstrap-schema--xref--bootstrap-keypath}
+
+|use_tls_server=false
+skip_tls_verify=true
+|* Sync Gateway will error non-secure server schemes (`couchbase:` or `ws:`) unless the opt-out is triggered
+* If a `ca_cert_path` is specified then only certificates from that CA will be accepted.
+* If *no* `ca_cert_path` is specified then only certificates from a trusted CA will be accepted, unless the opt-out is triggered (when no validation is performed)
+
+// |Launching Sync Gateway as Service
+// |The `serviceconfig.json` file uses `couchbases:` scheme, with `skip_tls_verify=true` by default  +
+//  SSLCert +
+// SSLKey
+// |skip_tls_verify
+// |Sync Gateway will allow non-secure server schemes (`couchbase:` or `ws:`) unless the opt-out option is set false (its default is `true` in this instance)
+
+// |Launching Sync Gateway at Command Line
+// | SSLCert +
+// SSLKey
+// |skip_tls_verify
+// |Sync Gateway will error non-secure server schemes (`couchbase:` or `ws:`) unless the opt-out option is true. +
+
+|===
+
+
+=== Launching Sync Gateway as Service
+The `serviceconfig.json` file uses the `couchbases:` scheme, with `skip_tls_verify=true` by default, to facilitate testing and development; no TLS validation is done.
+
+You can set TLS validation using the  following settings:
+
+* {bootstrap-schema--xref--bootstrap-ca-cert-path}
+* {bootstrap-schema--xref--bootstrap-certpath}
+* {bootstrap-schema--xref--bootstrap-keypath}
+* `skip_tls_verify=false`
+
+
+=== Launching Sync Gateway at Command Line
+
+You can set TLS validation using the  following command line flags:
+
+* -bootstrap-cert-path
+* -bootstrap-cacert-path
+* -bootstrap-key-path
+* -skip_tls_verify
```

Comment:
Would be `-bootstrap.server_tls_skip_verify`

Reply:
Done -- https://github.com/couchbase/docs-sync-gateway/pull/541